src/sourmash/sbt.py

```diff
@@ -183,6 +183,7 @@ def __init__(self, factory, *, d=2, storage=None, cache_size=None):
         if cache_size is None:
             cache_size = sys.maxsize
         self._nodescache = _NodesCache(maxsize=cache_size)
+        self.location = None
```

Comment:
Very minor comment: should this be 
```suggestion
        self._location = None
```
instead? Just to communicate it is an internal value and not to be accessed outside the class (and, if it is supposed to be accessed publicly, then make a property?)

Reply:
k, good idea.